inventory-service/src/main/java/ms/learnings/inventoryservice/dto/InventoryResponse.java

```diff
@@ -0,0 +1,15 @@
+package ms.learnings.inventoryservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class InventoryResponse {
```

Comment:
We want this to be unique based on skuCode

Reply:
Done